articles/create-a-schema.mdx

```diff
@@ -16,121 +16,225 @@ related_articles:
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/schemas/2024-03/CloudCannon-Documentation-Add-Button-Multiple-Schemas.png" alt="A closeup of the Collection Browser shows multiple options in the Add dropdown." title="Multiple schemas in the collection Add menu" type="ui-snippet"/>
 
-    If `true`, non-schema inputs are hidden in the Data Editor.
+<comp.Notice info_type="info">
+  Schemas create new files with the same file extension as the schema file. If you need to, you can reference schemas with different file extensions from the same collection to create different file types.
+</comp.Notice>
 
-    If `false`, non-schema inputs are visible in the Data Editor.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="icon" type_markdown="String">
-    Displayed in the add menu when creating new files, also used as the icon for collection files if no other preview is found. Must be a [Material Icon](https://material.io/resources/icons/) name. Defaults to `notes`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="name" type_markdown="String">
-    Displayed in the add menu when creating new files. Defaults to a formatted version of the key.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="new_preview_url" type_markdown="String">
-    Preview your unbuilt pages (e.g. drafts) to another page’s output URL. The Visual Editor will load that set preview URL and use the Data Bindings and Previews to render your new page without saving.
+## Update an existing schema
 
-    For example `new_preview_url: /about/` will load the `/about/` URL on new or unbuilt pages in the Visual Editor.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="path" type_markdown="String">
-    The path to the schema file. Relative to the root folder of the site. Required.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="reorder_inputs" type_markdown="Boolean">
-    Controls whether or not inputs are sorted to match the schema when editing existing files. Defaults to `true`.
+Schemas help you keep your collection files consistent. When you edit a file in a collection, CloudCannon checks that the inputs in the front matter match its defined schema. Updating a schema file by adding, removing, or reordering inputs will also update your collection files the next time you open them in an editing interface.
 
-    Non-schema inputs are ordered after schema inputs if not removed with `remove_extra_inputs`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="remove_empty_inputs" type_markdown="Boolean">
-    Sets the behavior when saving files with empty inputs. Defaults to `false`.
+<comp.Notice info_type="info">
+  CloudCannon does not update a collection file to match your schema until you open it in an editing interface.
+</comp.Notice>
```

Comment:
```suggestion
<comp.Notice info_type="info">
  CloudCannon does not update all existing collection files automatically when a schema changes. Each collection file will be updated when it is next opened in a non-source editing interface.
</comp.Notice>
```

Mainly trying to reword away from the ambiguous `it` (could read the original as `open it` referring to opening the schema in an editing interface, perhaps)